.github/workflows/core-macos.yml

```diff
@@ -8,32 +8,26 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
-      
+
       - name: configure OS
         run: |
           # os level stuff
-          pwd
-          ls
           set -x
-          brew install googletest
-      
+          brew install googletest llvm
```

Comment:
@burlen: do we need the change to ``llvm`` here? I was under the impression that we only needed llvm from homebrew if we wanted to compile with OpenMP. If the change isn't necessary, then I think it should be reverted here. I think there's value in keeping ``core-macos`` the simplest possible build configuration and relying on a separate build (``openmp-macos``) for the more complicated case with OpenMP.